apps/web/src/server/applications/case/s51/applications-s51.controller.js

```diff
@@ -136,6 +152,54 @@ export async function postApplicationsCaseEditS51Item({ body, params }, response
 
 	response.redirect('../properties');
 }
+/**
+ * Show s51 advice item
+ *
+ * @type {import('@pins/express').RenderHandler<{}, {}, ApplicationsS51ChangeStatusBody, {size?: string, number?: string}>}
+ */
+export async function changeAdviceStatus(request, response) {
+
+	const { errors: validationErrors, body } = request;
+
+	// @ts-ignore
+	const properties = await s51FolderData(request, response);
+
+	if (validationErrors) {
+		return response.render(`applications/components/folder/folder`, {
+			errors: validationErrors,
+			// @ts-ignore
+			...properties
+		});
+	}
+
+	/**
+	 * @type {{ id: number; }[]}
+	 */
+	const items = []
+	body.selectedFilesIds.forEach((/** @type {number} */ selectField) => {
+		items.push({id:  Number(selectField)});
+	})
```

Comment:
Just a stylistic improvement :)

```suggestion
        const items = body.selectedFilesIds.map(selectField => ({ id: Number(selectField) }));
```